docs/frontend/development.md

```
@@ -62,36 +81,48 @@ nvm install
+
+Open Google Chrome's Developer tools, and make sure you have cache disabled and correct settings to avoid stale content:
 
 :::info
 Instructions are for Google Chrome
 :::
 
-1. Disable cache by ticking the box in `Network` > `Disable cache`
+1. Disable cache by ticking the box in **Network** > **Disable cache**
 
 <p class='img'>
   <img src='/img/en/development/disable-cache.png' />
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)